docs/learn/learn-staking.md

```diff
@@ -1,324 +1,390 @@
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} backing get elected and become active. As a
-nominator, a minimum of
+candidates that it supports, and the network will automatically distribute the stake among
+validators in an even manner so that the economic security is maximized. In the next era, a certain
+number of validators having the most {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} backing
+get elected and become active. For more information about the election algorithm go to
+[this](learn-phragmen.md) page on the wiki or
+[this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#)
+research article. As a nominator, a minimum of
 {{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}
-is required to submit an intention to nominate. The nomination intents are placed in a semi-sorted
-list called [bags-list](https://github.com/paritytech/substrate/pull/9507).
+is required to submit an intention to nominate. Note that in NPoS the stake of both nominators and
+validators can be slashed. For an in-depth review of NPoS see
+[this](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) research article.
 
 :::caution Minimum Nomination to Receive Staking Rewards
 
 Although the minimum nomination intent is
-{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>, :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>, :kusama }}
+{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }},
 it does not guarantee staking rewards. The nominated amount has to be greater than
 [minimum active nomination](learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards),
 which is a dynamic value that can be much higher than
-{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>. :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>. :kusama }}
+{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}.
```

Comment:
This dynamic value depends on the amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} being staked, in addition to the selected nominations.

Reply:
Added